lms/envs/common.py

```diff
@@ -1861,6 +1861,12 @@
     'provider.oauth2',
     'oauth2_provider',
 
+    # We don't use this directly (since we use OAuth2), but we need to install it anyway.
+    # When a user is deleted, Django queries all tables with a FK to the auth_user table,
+    # and since django-rest-framework-oauth imports this, it will try to access tables
+    # defined by oauth_provider.  If those tables don't exist, an error can occur.
+    'oauth_provider',
```

Comment:
@feanil Contrary to what I said this morning, it looks like there will be migrations in this PR :(  As far as I know, the sysadmin dashboard for deleting users is the only part of the app that needs this, and it just needs the table to exist (it doesn't write to it).